File: koopey.web/src/app/models/base/audit.ts
const SHA256 = require("crypto-js/sha256");
import { Environment } from "src/environments/environment";
import { Base } from "./base";

export class Audit extends Base {

  public name: string = "";
  public type: string = "none";
  public description: string = "";
  public timeZone: string = Environment.Default.TimeZone;

  public isEmpty(): boolean {
    return this.name.length == 0 && this.type.length == 0 && this.description.length == 0 ? true : false;
  }

}

File: koopey.android/app/src/main/java/com/koopey/model/base/Audit.java
package com.koopey.model.base;

public class Audit extends Base {
    public String name = "";
    public String description = "";
    public String type = "";

    public boolean isEmpty() {
        return  name == null   || name.length() <= 0 || super.isEmpty() ? true : false;
    }
}
